README.md

```diff
@@ -9,4 +134,243 @@ The website for the Civic Tech Index
 *<ins>Example</ins>:
 `docker build -t webmasterimage .`
 `docker run -p 80:80 webmasterimage`
-In browser go to localhost:80*
\ No newline at end of file
+In browser go to localhost:80*
+
+### Build and serve the website locally
+
+This command starts a jekyll server locally. The server watches for changes to
+the source files and rebuilds and refreshes the site automatically in your browser.
+
+```bash
+docker-compose up
+```
+
+Now browse to http://localhost:4000
+
+### Tear down
+
+To stop and completely remove the jekyll server (i.e. the running Docker container):
+
+*(do this anytime Docker or jekyll configuration or other repository settings change)*
+
+```bash
+docker-compose down
+```
+
+To stop the server, but not destroy it (often sufficient for day-to-day work):
+
+```bash
+docker-compose stop
+```
+
+Bring the same server back up later with:
+
+```bash
+docker-compose up
+```
+<br>
+
+## Step 5: Change to a new branch
+
+Create a new branch for each issue you work on. Doing all your work on topic branches leaves your repository's main branch (named `gh-pages`) unmodified and greatly simplifies keeping your fork in sync with the main project.
+
+a) Check current branch
+
+The `git branch` command will let you know what branch you are in, and what branch names are already in use.
+
+```bash
+git branch
+```
+
+You will see a list of all of your branches. There will be a star (`*`) next to the branch that you are currently in. By default you should start on the `gh-pages` branch.
+
+b) Create a new branch where you will work on your issue
+
+The `git checkout` command will create and change to a new branch where you will do the work on your issue.  In git, the checkout command lets you navigate between different branches.  Using the `-b` flag you can create a new branch and immediately switch into it. 
+
+To create a new issue branch, and switch into it: 
```

Comment:
To create a new issue branch, and switch into it (use the following example, but make sure to read the commit below on customizing the name from "fix-logo-width-311" to your own:

Reply:
Please review